app/controllers/katello/api/v2/sync_plans_controller.rb

```diff
@@ -13,6 +13,7 @@ class Api::V2::SyncPlansController < Api::V2::ApiController
       param :sync_date, String, :desc => N_("start datetime of synchronization"), :required => true, :action_aware => true
       param :description, String, :desc => N_("sync plan description")
       param :enabled, :bool, :desc => N_("enables or disables synchronization"), :required => true, :action_aware => true
+      param :cron_logic, String, :desc => N_("Add custom cron logic for sync plan")
```

Comment:
I might would rename 'cron_logic' to 'cron_expression'.   That seems to be the standard name for it:  https://en.wikipedia.org/wiki/Cron#CRON_expression